winml/lib/Api/LearningModel.cpp

```diff
@@ -309,6 +309,43 @@ LearningModel::GetEngineFactory() {
   return engine_factory_.get();
 }
 
+void LearningModel::SaveToFile(const hstring& file_name) {
+  model_->SaveModel(file_name.c_str(), file_name.size());
```

Comment:
does "SaveModel" return any errors that need to be propagated?

Reply:
In theory all of the experimental models could use try catches.
I have not been adding them, but they need it holistically.
But specifically SaveModel returns void, so there isnt a return value to check.